deps.ffmpeg/40-aom.zsh

```diff
@@ -55,7 +55,6 @@ config() {
   args=(
     ${cmake_flags}
     -DBUILD_SHARED_LIBS="${_onoff[(( shared_libs + 1 ))]}"
-    -DAOM_TARGET_CPU="${arch}"
```

Comment:
Maybe this would work.
```sh
 case ${target} {
    macos-arm64) args+=(-DCONFIG_RUNTIME_CPU_DETECT=0 -DAOM_TARGET_CPU="${arch}") ;;
    windows-x*) args+=(-DCMAKE_TOOLCHAIN_FILE="build/cmake/toolchains/${target_config[cmake_arch]}-mingw-gcc.cmake")
  }
```